harness/attributes/common.yml

```diff
@@ -236,37 +236,51 @@ attributes.default:
     mountVolumesOnConsole: true # possible to disable, but may lead to unexpected concequences with app init/migrate
 
     elasticsearch:
-      enabled: true
       accessMode: ReadWriteOnce
+      claimName: "{{ $.Release.Name }}-elasticsearch-pv-claim" # legacy naming
       size: 2Gi
+      services:
+        - elasticsearch
```

Comment:
I'm honestly not keen on this bit although necessary, looks messy structure.

The only compromise I can think of is have two concepts of PVCs, service PVCs and project PVCs, with the former living in the service config.

it was a bit easier concept on the fully helm value project, since each project defined their pvc requirements like this, so was implicitly enabled.